tests/integration/test_route_update.py

```
@@ -0,0 +1,271 @@
+        # Pre-populate device with old data
+        mock_device.states['train1Destination'] = 'Old Destination'
+        mock_device.states['train2Destination'] = 'Old Destination'
+
+        api_key = "test_api_key"
+        network_url = "https://lite.realtime.nationalrail.co.uk/OpenLDBWS/wsdl.aspx"
+
+        from mocks.mock_darwin import create_mock_darwin_session
+        mock_session = create_mock_darwin_session("empty")
+
+        with patch('plugin.DarwinLdbSession', return_value=mock_session):
+            result = plugin.routeUpdate(
```

Ruff (0.14.14)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)